starport/pkg/gomodulepath/gomodulepath.go

```diff
@@ -42,20 +47,37 @@ func Parse(rawpath string) (Path, error) {
 		return Path{}, err
 	}
 	p := Path{
+		AppPath: appPath,
```

Comment:
```suggestion
```

Reply:
We need the app path field into the Path object. It's used in too many places, is there any other suggestion for we can remove it?

<img width="981" alt="Screen Shot 2021-09-02 at 12 19 55" src="https://user-images.githubusercontent.com/2406457/131870929-24857219-5b72-4aac-be2d-def80c610e61.png">

Reply:
the problem with this is AppPath is not relevant for the Parse func. because it's passed as an argument to Parse then placed into Path, but not used by the parsing logic anyhow. I understand that passing it through like this makes the usage easier but it's not convenient to pass arguments to funcs if they are not actually used by the func. it's an unnecessary dependency.